### DISCUSSION TITLE
This Serverless Function has timed out.

### DISCUSSION THREAD

**Discussion by hugotox** [vercel/community#5733] [Help] [answered]
### Summary

My website www.flystickmonkey.com has been working without issues until this morning. It just shows the timeout error page, no error in the logs, and the usage is way below threshold. 
Any advice?
<img width="543" alt="image" src="https://github.com/vercel/community/assets/1758062/573c1616-e82f-4de0-97a6-0d9de11dca78">
<img width="325" alt="image" src="https://github.com/vercel/community/assets/1758062/d77a75b6-f1ae-4bbf-a723-2f5aa477d310">
<img width="1242" alt="image" src="https://github.com/vercel/community/assets/1758062/13827835-057e-4b3a-a4d2-5fe08a6cada6">

### Steps to Reproduce

I can't tell, it just stopped working www.flystickmonkey.com

**Comment by versecafe** [comment, marked as answer]
@hugotox On the free tier serverless functions have a max duration of 10 seconds, if you call an outside API or make a DB call which is hanging or has grown in run time that could drive the error, this could be just a temporary issue with an API you're calling as the site seems to be fine currently for the `/` Route.

**Reply by hugotox** [reply]
Thanks for responding, but this was an issue with Vercel, all my backend APIs were up and running, and my page came back online after around 10 minutes.
